kafka-ui-react-app/src/components/common/NewTable/Table.tsx

```diff
@@ -160,6 +157,9 @@ const Table: React.FC<TableProps<any>> = ({
       pagination: getPaginationFromSearchParams(searchParams),
       rowSelection,
     },
+    getRowId: (originalRow, index) => {
+      return originalRow.name ? `${originalRow.name}` : `${index}`;
```

Comment:
Do we need the to put it in a string quotation string  literals here for the `${originalRow.name}`?

Reply:
I think not, it is obviously a string value)

Reply:
Isn't it already a name?

Reply:
Yes, it's a string, so we do not need quotation marks - update in a moment